espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/w3c/models/InputSource.java

```diff
@@ -106,14 +107,11 @@ public PointerType getPointerType(){
         private Long duration; // time in milliseconds
         private String origin; // origin; could be viewport, pointer or <ELEMENT_ID>
         private Integer button; // Button that is being pressed. Defaults to 0.
-        private Long x; // x coordinate of pointer
-        private Long y; // y coordinate of pointer
+        private long x; // x coordinate of pointer
```

Comment:
does this mean that the serializer is supposed to throw an exception if the coordinate is not provided?

Reply:
No, it should still accept it if the coordinate is not provided. If none is provided it would default to 0 which is okay.

Reply:
I think than both coordinates are required according to W3C